### PR TITLE
Add support for Orca tf estimator save/load weights

### PR DESCRIPTION
**Pull request by jenniew**
Add orca.tf.Estimator APIs:
```
def save_keras_model_weights(self, filepath, overwrite=True, save_format=None)
def load_keras_model_weights(self, filepath, by_name=False)
@staticmethod
def load_keras_model(path)
```

### CODE REVIEW

File: pyzoo/zoo/orca/learn/tf/estimator.py [hunk 1]
```diff
@@ -197,6 +198,43 @@ def save_keras_model(self, path, overwrite=True):
...
+        """
+        from tensorflow.python.keras import models
+
+        def load_func(file_path):
+            return models.load_model(file_path)
+
+        model = load_from_file(load_func, path)
+        return Estimator.from_keras(keras_model=model)
+
+    def save_keras_model_weights(self, filepath, overwrite=True, save_format=None):
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf/estimator.py:217]: `save_keras_weights`

File: pyzoo/zoo/orca/learn/tf/estimator.py [hunk 2]
```diff
@@ -197,6 +198,43 @@ def save_keras_model(self, path, overwrite=True):
...
+        Save tensorflow keras model weights in this estimator.
+        :param path: keras model weights save path.
+        :param overwrite: Whether to silently overwrite any existing file at the target location.
+        :param save_format: Either 'tf' or 'h5'. A `filepath` ending in '.h5' or
+            '.keras' will default to HDF5 if `save_format` is `None`. Otherwise
+            `None` defaults to 'tf'.
+        """
+        raise NotImplementedError()
+
+    def load_keras_model_weights(self, filepath, by_name=False):
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf/estimator.py:228]: `load_keras_weights`

Comment (jenniew) [reply pyzoo/zoo/orca/learn/tf/estimator.py]: updated

File: pyzoo/zoo/orca/learn/tf/estimator.py (hunk 1 above)
Comment (jenniew) [reply pyzoo/zoo/orca/learn/tf/estimator.py]: updated

**Comment by jenniew** [comment]
Jenkins pass: http://10.239.47.210:18888/job/ZOO-PR-Validation/4562/

**Pull request merged**